tests/disruption_helpers.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+
+
+logger = logging.getLogger(__name__)
+
+POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
+
+
+class Disruptions:
+    """
+    This class contains methods of disrupt operations
+    """
+    resource = None
+    resource_obj = None
+    resource_count = 0
+
+    def set_resource(self, resource):
+        self.resource = resource
+        if self.resource == 'mgr':
+            self.resource_obj = pod.get_mgr_pods()
+        if self.resource == 'mon':
+            self.resource_obj = pod.get_mon_pods()
+            self.resource_count = 3
+        if self.resource == 'osd':
+            self.resource_obj = pod.get_osd_pods()
```

Comment:
@ebenahar since wait_for_resource() is expecting "resource_count=self.resource_count" , how will OSD pod delete be handled ? 

Sorry If i missed anything.

Reply:
You're absolutely right :) Will be handled in an upcoming commit by the code in my comment above